.goreleaser.yaml

```diff
@@ -28,7 +30,7 @@ checksum:
   name_template: 'checksums.txt'
 
 snapshot:
-  name_template: "{{ incpatch .Version }}-next"
+  name_template: "{{ incpatch .Version }}-dev-{{ .ShortCommit }}"
```

Comment:
how is this snapshot used, are you sure you want to hard-code dev here?

Reply:
Yeah, this becomes the version strung but only if you do `goreleaser --snapshot` to generate a snapshot build. When you actually release, it doesn't use this, it uses the actual version string (`v1.2.3` or whatever)